script.js

```diff
@@ -0,0 +1,93 @@
+/* ---------- Slideshow ---------- */
+let current_slide = 0;
+let slideNodes = [];
+
+function showSlide(n) {
+  if (!slideNodes.length) return;
+  // wrap-around
+  if (n >= slideNodes.length) current_slide = 0;
+  else if (n < 0) current_slide = slideNodes.length - 1;
+  else current_slide = n;
+
+  for (let i = 0; i < slideNodes.length; i++) {
+    slideNodes[i].style.display = (i === current_slide) ? "block" : "none";
+  }
+}
+
+function previous() { showSlide(current_slide - 1); }
+function next()     { showSlide(current_slide + 1); }
+
+function initSlideshow() {
+  slideNodes = Array.from(document.querySelectorAll(".slideshow .slideshow_img"));
+  showSlide(0);
+}
+
+/* ---------- To-Do with localStorage ---------- */
+let items = [];
```

Comment:
By putting everything in the same file, the variables items and STORAGE_KEY get created on your slideshow page too, which makes your page heavier to load. It should be in a separate .js file.